distributed/deploy/tests/test_spec_cluster.py

```diff
@@ -81,6 +81,15 @@ def test_loop_started():
     )
 
 
+@pytest.mark.asyncio
+async def test_repr():
+    class MyCluster(SpecCluster):
+        pass
+
+    async with MyCluster(..., asynchronous=True) as cluster:
```

Comment:
Ah, blind copy-paste issue I think?  I used `...` to mean "I haven't thought about what should go in here too much, but you could probably copy-paste from similar tests"

Reply:
Ha my bad! I assumed that was some clever shorthand for defaulting everything that I hadn't seen before.